### DISCUSSION TITLE
Tips for Freshers and College Students

### DISCUSSION THREAD

**Discussion by AbhijithGanesh** [EddieHubCommunity/support#2364] [Community Call]
What are all the things a fresher can do to have a good blooming start to their career and college life.

**Comment by srisatya12** [comment]
Hey @AbhijithGanesh participate in Open Source programs. There are many Open Source programs where you can kickstart your open Source Contributions like in october there's HactoberFest and there are many other Open Source Programs. if you don't want to miss these oppurtunities. Be active on Twitter and Linkedin. Join some amazing communities. Participate in some Hackathons and events.
